### PR TITLE
VtC: Recording flow UI

### PR DESCRIPTION
**Pull request by momo-ozawa**
Part of https://github.com/Automattic/wordpress-mobile/issues/102

## Description
- Match design specs (ref: p1HpG7-sJz-p2)

## How to test
- Verify that the recording flow UI matches the design specs (ref: p1HpG7-sJz-p2)

https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/7a6a994d-d708-4806-8e2b-6f306fb56a35

## Regression Notes
1. Potential unintended areas of impact

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

Testing checklist:
- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] VoiceOver.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] iPhone and iPad. 
- [ ] Multi-tasking: Split view and Slide over. (iPad)

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift [hunk 1]
```diff
@@ -158,13 +179,20 @@ private struct VoiceToContenProcessingView: View {
     var body: some View {
         VStack {
             Spacer()
-            ProgressView()
+            if #available(iOS 17.0, *) {
+                ProgressView()
+                    .controlSize(.extraLarge)
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift]: (nit) I'd suggest using `.large` (iOS 15+)

File: WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift [hunk 2]
```diff
@@ -128,26 +131,44 @@ private struct VoiceToContentRecordingView: View {
...
                 if #available(iOS 17.0, *) {
                     Image(systemName: "waveform")
-                        .font(.system(size: 80, weight: .medium))
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: 80)
                         .foregroundStyle(Color(uiColor: .brand))
                         .symbolEffect(.variableColor, isActive: true)
+                } else {
+                    Image("audio-waveform")
```
Comment (kean) [inline WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift]: (nit) `Image(systemName: "waveform")` is available on iOS 15. The only part that is not is `symbolEffect`.

**Review by kean — APPROVED** [review]

File: WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift (hunk 2 above)
Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift]: Fixed! [bdd9077](https://github.com/wordpress-mobile/WordPress-iOS/pull/23258/commits/bdd9077951e9258626b59ae667f10fa549332a03)

File: WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift (hunk 1 above)
Comment (momo-ozawa) [reply WordPress/Classes/ViewRelated/Voice/VoiceToContentView.swift]: Fixed! [2283d13](https://github.com/wordpress-mobile/WordPress-iOS/pull/23258/commits/2283d133a07387109e29508d70d34b8d795535cd)

**Pull request merged**